rfcs/0024-ckb-genesis-script-list/0024-ckb-genesis-script-list.md

```diff
@@ -0,0 +1,216 @@
+---
+Number: "0024"
+Category: Informational
+Status: Final
+Author: Dylan Duan
+Organization: Nervina Labs
+Created: 2020-05-21
+---
+
+# CKB Genesis Script List
+
+## Abstract
+
+Genesis scripts are the smart contracts built by the CKB team and set in the CKB genesis block. Genesis scripts provide core functions like (e.g. [SECP256k1/blake160](#secp256k1blake160) and [Nervos DAO](#nervos-dao)). This document presents the information of all Nervos CKB genesis scripts, including a brief introduction and _code_hash_, _hash_type_, _out_point_(_tx_hash_ and _index_), _dep_type_ on mainnet Lina and testnet Aggron.
```

Comment:
> Genesis scripts are the smart contracts built by the CKB team and set in the CKB genesis block.

Strictly speaking `TYPE_ID` is a `builtin` script, and it is not set in the CKB genesis block, it is directly implemented in CKB (not run in CKB-VM) in Rust.

Reply:
I suggest adding the above comment to `type id` section.

Reply:
> > Genesis scripts are the smart contracts built by the CKB team and set in the CKB genesis block.
> 
> Strictly speaking `TYPE_ID` is a `builtin` script, and it is not set in the CKB genesis block, it is directly implemented in CKB (not run in CKB-VM) in Rust.

I will add the comment to `Type ID` section.